recipes/picrust/meta.yaml

```diff
@@ -1,36 +1,36 @@
 {% set name = "picrust" %}
-{% set version = "1.1.3" %}
+{% set version = "1.1.4" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/picrust/picrust/archive/v{{ version }}.tar.gz
-  sha256: 7538c8544899b8855deb73a2d7a4ccac4808ff294e161530a8c8762d472d8906
+  sha256: ebfca914002ceb6d35376370e075379427d24e0c7711d46597afb52d4738c880
 
 build:
-  number: 2
-  skip: True # [ not py27 ]
+  number: 0
+  noarch: generic
```

Comment:
Please change this to `noarch: python`, though honestly there's no benefit of that over the previous `skip: True  # [no py27]`.

Reply:
Changed.